dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -48,14 +48,27 @@ public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGit
         Multimap<String, String> pathToDockerfilesInParentRepo = ArrayListMultimap.create();
 
         Set<Map.Entry<String, JsonElement>> imageToTagStore = parseStoreToImagesMap(ns.get(Constants.STORE));
+        Integer gitApiSearchLimit = ns.get(Constants.GIT_API_SEARCH_LIMIT);
+        List<IOException> exceptions = new ArrayList<>();
+        Map<String, Boolean> orgsToIncludeInSearch = new HashMap<>();
+        orgsToIncludeInSearch.put(ns.get(Constants.GIT_ORG), true);
```

Comment:
What if `GIT_ORG` wasn't specified? Does this work properly?

Reply:
> @avimanyum Thanks! This is awesome! Could you edit the description of what the net change of this PR does? It seems to appear that it will potentially exclude one org with the top hits but I haven't given it a super deep look.

Added the description [here](https://github.com/salesforce/dockerfile-image-update/pull/276#issue-757738367). Please let me know if that looks accurate. Is there some other place you wanted me to add this to? A

Reply:
Thanks for catching this. Will add a check here

Reply:
Were you still going to add a check?

Reply:
Yes, added a check.